### ISSUE TITLE
Could I using your dropout without training the model from scratch? 

### ISSUE THREAD

**Issue by wangpichao** [ooibc88/dropout#1]
I notice in your experiments, you train the whole networks from end2end on several datasets. You know, in most computer vision applications, usually, we will finetune the ResNet or other networks. I wonder whether we could insert your dropout into available ResNet, for example resnet50, and initialize the networks from pretained models and finetune  it?

**Comment by solopku** [comment]
We did not try this in our experiments, but to our best knowledge, you can initialize the network from pretrained models and then finetune it with dropout.
One practical suggestion is that you may try gradually (e.g., linearly) increasing the dropout rate from 0.0 to the final rate (typically a small one, e.g., 0.1), to avoid training instability in the beginning of finetuning.
In addition, Dropout2d is preferable to standard dropout in most vision tasks.
